src/components/map/MapService.js

```diff
@@ -720,7 +720,10 @@ goog.require('ga_urlutils_service');
                 },
                 // Temporary until https://github.com/openlayers/ol3/pull/4964
                 // is merged upstream
-                cacheSize: 2048 * 3,
+                // Workaround: Set a cache size of zero when layer is
+                // timeEnabled see:
+                // https://github.com/geoadmin/mf-geoadmin3/issues/3491
+                cacheSize: layer.timeEnabled ? 0 : 2048 * 3,
```

Comment:
According to the old comment this cache size is not needed anymore (ol3 update has been merged), so maybe we could set to 0 everytime. 

Anyway you can remove the old comment.